src/main/java/com/courtcanva/ccfranchise/controllers/FranchiseeController.java

```diff
@@ -34,8 +36,15 @@ public FranchiseeAndStaffDto signUpFranchiseeAndStaff(@RequestBody @Valid Franch
 
     @PostMapping("/{franchiseeId}/service_areas")
     @ResponseStatus(HttpStatus.CREATED)
-    public SuburbListGetDto addDutyAreas(@RequestBody @Valid SuburbListPostDto suburbListPostDto, @PathVariable(value = "franchiseeId") Long franchiseeId) {
+    public SuburbListGetDto addDutyAreas(@RequestBody @Valid SuburbListPostDto suburbListPostDto,
+                                         @PathVariable(value = "franchiseeId") Long franchiseeId) {
         return franchiseeService.addDutyAreas(suburbListPostDto, franchiseeId);
     }
 
+    @PostMapping("/{franchiseeId}/accept-orders")
```

Comment:
check the uri used in above, it is `service_area` and this one is `accept-orders`, please keep consistency.